components/dashboard/src/teams/TeamSettings.tsx

```diff
@@ -174,7 +155,7 @@ export default function TeamSettings() {
             <ConfirmationModal
                 title="Delete Team"
                 buttonText="Delete Team"
-                buttonDisabled={teamNameToDelete !== team!.name}
```

Comment:
Fixing a potential runtime bug (bumped into it during dev) - generally we should avoid using `!` to suppress TS errors, especially in render logic.